docs/gestor_tareas.md

```diff
@@ -0,0 +1,83 @@
+# Gestor de Tareas
+
+A diferencia de otros lenguajes, como Go no incorpora un gestor de tareas nativo necesitamos recurrir a herramientas externas para automatizar pasos como:
+- Compilación
+- Ejecución de tests
+- Formateo y lint
+- Ejecutar tareas previas a despliegue o CI/CD
+
+## Criterios para designar el gestor de tareas
+1. La herramienta debe estar actualizada como máximo en los últimos 6 meses, de esta forma reduciremos la deuda técnica.
+2. Capacidad para automatizar comandos del toolchain (build, test, vet...), independientemente de si la herramienta usa sintaxis propia o genera llamadas externas.
```

Comment:
De veras, no pongáis esto. Esto lo van a hacer todas. Si lo que quieres decir es que si está en el toolchain, eso que te ahorras, dilo así.